concourse/scripts/check-clang-format.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+# Due to the limitation of concourse git/github-pr resource, it is difficult to
+# only check the format of the git diff. So all the source code are being
+# checked.
+
+echo "Disable clang-format"
```

Comment:
XD
Maybe we can just remove clang-format for this repo. I don't think we will ever use it for IP4R.